packages/@sanity/cli/test/shared/globalSetup.ts

```diff
@@ -79,16 +79,18 @@ function prepareStudios() {
       const destinationPath = path.join(studiosPath, version)
       const customDocStudioPath = path.join(studiosPath, `${version}-custom-document`)
 
-      await mkdir(destinationPath, {recursive: true})
```

Comment:
I assume this is because the `copy()` function below creates it?

Reply:
Good question - looks like cpx [creates](https://github.com/mysticatea/cpx/blob/master/lib/utils/copy-file.js#L104C10-L104C10) it, but I reverted this line to make sure we don't depend on undocumented/unspecified behavior.